Added Raspberry pi support

spaniakos opened the pull request:
hello there, the library is functioning correctlly to raspberry pi as well.

spaniakos committed:
added RPi support

spaniakos committed:
new readme

spaniakos committed:
Update README.md

spaniakos committed:
Update README.md

spaniakos committed:
Update Makefile

spaniakos committed:
Update Makefile

spaniakos committed:
char >> const char

spaniakos committed:
Config file + example Changes

spaniakos committed:
added time measurement

spaniakos committed:
RPi fixes

spaniakos committed:
Merge remote-tracking branch 'upstream/master'

spaniakos committed:
Added

spaniakos committed:
added md5 and hmac

this is a test code

spaniakos committed:
Update MD5_config.h

spaniakos committed:
added md5 and hmac md5

experimantal code

spaniakos committed:
Update MD5_Hash.ino

spaniakos committed:
Update MD5_Hash.cpp

spaniakos committed:
Update README.md

spaniakos committed:
added md5 and hmac

testing code

spaniakos committed:
Update MD5.cpp

spaniakos committed:
Update MD5.cpp

spaniakos committed:
Hmac added and tested NO RPI YET

spaniakos committed:
examples hmac

spaniakos committed:
HMAC md5

spaniakos committed:
better clean

spaniakos committed:
Hmac update

spaniakos committed:
Readme update

spaniakos commented:
now the library supports HMAC

spaniakos committed:
changes for Documenation

spaniakos committed:
added examples_Rpi dirR

spaniakos committed:
changes

spaniakos committed:
Update README.md

tzikis commented:
Holy moly, Giorgo thanks for the updates.

I am very tempted to just accept the changes, but it goes against my spirit of reviewing changes like that and, since I know that people looking for this will probably also be able to find your fork, I am keeping the ticket open until I can dedicate the time to review it properly. I hope you don't mind

spaniakos commented:
Kanena thema.

:-) me ths isixua sou
On Feb 25, 2015 2:25 AM, "Vasilis Georgitzikis" <notifications@github.com>
wrote:

> Holy moly, Giorgo thanks for the updates.
>
> I am very tempted to just accept the changes, but it goes against my
> spirit of reviewing changes like that and, since I know that people looking
> for this will probably also be able to find your fork, I am keeping the
> ticket open until I can dedicate the time to review it properly. I hope you
> don't mind
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tzikis/ArduinoMD5/pull/8#issuecomment-75887860>.
>

spaniakos commented:
Thelw na kanw kai mia allagh akoma .
To digest na eibgebej by default san digest kai me ena flag na mporei na
gurnaei san string opos einai twra.
On Feb 25, 2015 8:41 AM, "spaniakos" <spaniakos@gmail.com> wrote:

> Kanena thema.
>
> :-) me ths isixua sou
> On Feb 25, 2015 2:25 AM, "Vasilis Georgitzikis" <notifications@github.com>
> wrote:
>
>> Holy moly, Giorgo thanks for the updates.
>>
>> I am very tempted to just accept the changes, but it goes against my
>> spirit of reviewing changes like that and, since I know that people looking
>> for this will probably also be able to find your fork, I am keeping the
>> ticket open until I can dedicate the time to review it properly. I hope you
>> don't mind
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/tzikis/ArduinoMD5/pull/8#issuecomment-75887860>.
>>
>

tzikis commented:
cool, but let's keep it in english since this is a public issue ;)

spaniakos commented:
Sorry i forgot .

I am in spain for some more days and the time was way to early :-)
On Feb 25, 2015 10:24 AM, "Vasilis Georgitzikis" <notifications@github.com>
wrote:

> cool, but let's keep it in english since this is a public issue ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tzikis/ArduinoMD5/pull/8#issuecomment-75929485>.
>

jwd83 committed:
Update HMAC_MD5.ino

The timing calculation shouldn't take into account the Serial.print("done. ("); call.

spaniakos committed:
Merge pull request #2 from jared0x90/patch-1

The timing calculation shouldn't take into account the Serial.print("done. ("); call.

spaniakos committed:
LICENSE.txt

beicnet commented:
Nice, after 5-6 years still not reviewed and merged! :D
